aries-site/src/examples/templates/ScreenContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, ResponsiveContext } from 'grommet';
+import { SidebarExample } from '.';
+
+export const ScreenContainer = ({ mobile, ...rest }) => (
+  <ResponsiveContext.Provider value={mobile && 'small'}>
+    <ResponsiveContext.Consumer>
+      {size => (
```

Comment:
Can we pull `size` from `useContext`?

Reply:
Because we are establishing the ResponsiveContext.provider value in this file, we can't use the hook.